src/bot.ts

```diff
@@ -13,39 +14,104 @@ const channelId = process.env.TELEGRAM_CHANNEL_ID as string
 assert(token, 'TELEGRAM_TOKEN env var is required')
 assert(channelId, 'TELEGRAM_CHANNEL_ID env var is required')
 
-const MESSAGE_GO_TO_CHANNEL = `Hey, for now I prefer to talk in dFusion channel.
-
-Please, go to t.me/dFusionPoC to get notified when there's a new standing order.
-
-Also, here are some links you might find useful:
-- https://github.com/gnosis/dex-contracts: dFusion Smart Contracts
-- https://github.com/gnosis/dex-research: dFusion Research
-- https://github.com/gnosis/dex-services: dFusion services`
-
 const bot = new TelegramBot(token, {
   polling: true
 })
 
-// Matches "/echo [whatever]"
-bot.onText(/\/echo (.+)/, (msg: Message, match: RegExpExecArray | null) => {
-  const resp = match ? match[1] : 'N/A'
+bot.onText(/\/(\w+) ?(.+)?/, (msg: Message, match: RegExpExecArray | null) => {
+  log.debug('New command: %o', msg)
 
-  log.debug('Received an echo message: %s, answering in 3s. %s, %o', resp, msg, match)
-  const text = 'This is a new order: ' + resp
-  log.debug('Writing into the channel', text)
-  bot.sendMessage(channelId, text)
+  _runCommand(msg, match).catch(error => {
+    log.error('Error running command for message: %o', msg)
+    log.error(error)
+  })
 })
 
 // Listen to any message
 bot.on('message', (msg: Message) => {
-  const chatId = msg.chat.id
-  log.debug('Received msg: %o', msg)
-
-  bot.sendMessage(chatId, MESSAGE_GO_TO_CHANNEL)
+  const isCommand = msg.text && msg.text.startsWith('/')
+  if (!isCommand) {
+    log.debug('Received msg: %o', msg)
+    _helpCommand(msg)
+  }
 })
 
 onShutdown(() => {
   log.info('Bye!')
 })
 
+async function _runCommand (msg: Message, match: RegExpExecArray | null) {
+  const command = match ? match[1] : ''
+  switch (command) {
+    case 'start':
+    case 'help':
+      await _helpCommand(msg)
+      break
+
+    case 'about':
+      await _aboutCommand(msg)
+      break
+
+    default:
+      await bot.sendMessage(msg.chat.id, "I don't recognize that command! You can use this other one instead: /help")
+  }
+}
+async function _helpCommand (msg: Message) {
+  const fromUser: User | undefined = msg.from
+  bot.sendMessage(
+    msg.chat.id,
+    `${fromUser ? 'Hi ' + fromUser.first_name : 'Hi there'}!
+    
+I don't talk much for now. I just notify every new order in dFusion channel.
+Please, go to t.me/dFusionPoC to get notified on every new order.
+
+Also, you can ask about me by using the command: /about`
+  )
+}
+
+async function _aboutCommand (msg: Message) {
+  const [blockNumber, networkId, nodeInfo] = await Promise.all([
+    dfusionRepo
+      .getBlockNumber()
+      .then(_toString)
```

Comment:
consider using `.then(String)` as it's [more reliable](https://devdocs.io/javascript/global_objects/string#String_conversion)

Reply:
uhhh wait but 

```
String(null)
// "null"
```

is that what we want?

Reply:
I think 'null' there it's fine, just to show in the about message. See the picture in description

Reply:
I guess, normally we will get the block